src/app/devices/devices.component.html

```diff
@@ -100,8 +100,15 @@ <h3 [fxShow]="isNoData()" fxLayoutAlign="center">No Devices</h3>
                     </div>
                 </mat-cell>
             </ng-container>
-
-            <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
+            <ng-container matColumnDef="notification">
+                <mat-header-cell *matHeaderCellDef fxFlex="50px"></mat-header-cell>
+                <mat-cell  *matCellDef="let element" fxFlex="50px">
+                    <mat-icon style="color: #2a8231;" matTooltip="Power action sent successfully" *ngIf="checkStatus(element) === 'done' ">check_circle_outline</mat-icon>
```

Comment:
shouldn't use custom colors here, leverage our color pallete. success should be `color=primary` the warning and error should both be `color=warn`. Also, please use the non-outlined versions of the icons.